presto-docs/src/main/sphinx/release/release-0.285.rst

```diff
@@ -0,0 +1,75 @@
+=============
+Release 0.285
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix `min_by(x, y, n) / max_by(x, y, n)` functions to ensure `n` is unique.
+* Fix LIKE pattern to handle multi-byte characters. :pr:`21578`
+* Improve latency of HBO. :pr:`21297`
+* Improve HBO to track partial aggregation statistics. :pr:`21160`
+* Improve PushPartialAggregationThroughExchange to use partial aggregation statistics when available. This extends to multi-key aggregations.
+* Improve union node by removing the empty inputs.
+* Improve HBO latency by skipping retry if it times out. :pr:`21308`
+* Improve join-reordering to work with non-simple equi-join predicates.
+* Improve :doc:`REST API</rest>` by providing links.
```

Comment:
```suggestion
* Improve :doc:`REST API</rest>` documentation by providing links.
```
Suggest adding the word "documentation" to the three doc entries - with the loss of the heading the context for these, when grouped into General Changes, feels missing. It's not immediately obvious when reading the page that these three entries are docs-only changes and didn't affect code.

Reply:
Good catch!